### ISSUE TITLE
[ENH] Prophet fitting should be by default non-verbose

### ISSUE THREAD

**Issue by fkiraly** [alan-turing-institute/sktime#1733]
When running `fit` of prophet with default parameters, following output is generated:

```
Initial log joint probability = -52.8485
Iteration  1. Log joint probability =    45.5064. Improved by 98.3549.
Iteration  2. Log joint probability =    58.4846. Improved by 12.9782.
Iteration  3. Log joint probability =    59.1316. Improved by 0.647006.
Iteration  4. Log joint probability =     59.147. Improved by 0.0154346.
Iteration  5. Log joint probability =     59.147. Improved by 2.3214e-005.
Iteration  6. Log joint probability =     59.147. Improved by 1.05452e-006.
Iteration  7. Log joint probability =    59.1471. Improved by 7.16411e-006.
Iteration  8. Log joint probability =    59.1511. Improved by 0.00408052.
Iteration  9. Log joint probability =    59.1511. Improved by 3.5126e-006.
Iteration 10. Log joint probability =    59.1511. Improved by 2.28879e-007.
Iteration 11. Log joint probability =    59.1511. Improved by 5.03879e-009.
```

However, the default should be non-verbose (with an option to turn this on).

**Comment by Carlosbogo** [comment]
Hi, I would like to work on this. Just to be sure, the idea would be to add an option to `fbprophet.Prophet` to choose if you want to generate the output you mention, right?

**Comment by fkiraly** [comment]
@Carlosbogo, all yours, thanks!

Regarding your question: yes/no.

The class already seems to have an argument `verbose`, but using `fit` with default parameters still seems to generate the printout.

There are first some questions to answer before making any changes:
* can you verify that it also generates the output for you. If not, then there is some version/dependency magic going on that may be hard to track down.
* if yes, then either: the current `verbose` argument has the wrong default (it should be non-verbose), or it is not correctly linked to the wrapped `prophet` estimator's verbosity

In terms of actions once the questions are answered:
* if we can't verify this, not sure what to do. Perhaps more detective work is needed
* if it's the wrong default, flip it
* if it's not correctly linked, that's a bug, fix it

**Comment by markvrma** [comment]
Upon running `fit` of `fbprophet.Prophet` with default parameters the output I got was,

```
INFO:prophet:Disabling weekly seasonality. Run prophet with weekly_seasonality=True to override this.
INFO:prophet:Disabling daily seasonality. Run prophet with daily_seasonality=True to override this.
Prophet()
```

In the `Prophet` constructor, `verbose` is initially set to 0, there was no difference in output upon changing it to 1/True.

Also I just wanted to confirm if `sktime` adapter includes all adapters in `forecasting.base.adapters`?

**Comment by fkiraly** [comment]
> Also I just wanted to confirm if `sktime` adapter includes all adapters in `forecasting.base.adapters`?

Yes, I think all adapters for forecasting packages (interface-to-interface) are in there.

**Comment by liuxiangchao369** [comment]
m.fit(df.loc[idx, ['ds', 'y', 'cap', 'floor']], verbose=False)

**Comment by liuxiangchao369** [comment]
m.fit(df.loc[idx, ['ds', 'y', 'cap', 'floor']], verbose=False)       # set verbose=False can hide the log
